cmd/kube-apiserver/app/server.go

```diff
@@ -430,6 +430,7 @@ func BuildGenericConfig(s *options.ServerRunOptions) (*genericapiserver.Config,
 			versionedInformers.Core().V1().Services().Lister(),
 		)
 	}
+	genericConfig.SharedInformerFactory = versionedInformers
```

Comment:
#50661 is just that victim.

I looked up that implementation and found `genericConfig.SharedInformerFactory` just stay unchanged since init. I think it's better to add this line here instead of [aggregator.go#L47-L85](https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-apiserver/app/aggregator.go#L47-L85).